tools/system-health-check/health_check.py

```diff
@@ -0,0 +1,210 @@
+import requests
+import json
+import kh_client
+import os
+
+prometheus = os.getenv("PROMETHEUS_ENDPOINT", "http://nopo11y-stack-kube-prometh-prometheus:9090")
+namespace = os.getenv("NAMESPACE","default")

```

Comment:
Add support for multiple namespaces

Reply:
As discussed, this will support one namespace for now